src/Opulence/Validation/Rules/Rules.php

```diff
@@ -382,6 +382,18 @@ public function required() : self
         return $this;
     }
 
+    /**
+     * Marks a field as to be validated even if empty
+     *
+     * @return self For method chaining
+     */
+    public function validateEmpty() : self
+    {
+        $this->isRequired = true;
```

Comment:
Instead of reusing `$this->isRequired`, could you add a new field to indicate that we are validating empty values?  My concern is that we're not really saying the field is required - we're just saying we should check it regardless of whether or not it's empty.  Maybe a new bool field like `private $validateEmptyFields = false`, and check that when determining whether or not to validate an empty value.